### ISSUE TITLE
Reverse cancel & submit buttons primary class usage

### ISSUE THREAD

**Issue by Shelob9** [DavidCramer/pods-frontier#28]
Same reason as I gave for #22. Lot's of buttons with the primary class that do cancel or delete and buttons for submit or add that do not have primary class. It's playing against the established usage pattern in WordPress admin.

**Comment by DavidCramer** [comment]
Agreed. I'm used to highlighting buttons that do dangerous actions like delete. But in WordPress it's opposite. 

I'll make changes.

**Comment by Shelob9** [comment]
You may be right about highlighting dangers, but going against an established pattern is a bigger danger. BTW I realize now that this same issue was bothering me in Caldera Forms, I kept removing things inadvertently and not realizing where they went.

**Comment by DavidCramer** [comment]
Ye I realized that and am working on it.

-----Original Message-----
